DatosAssignment2_[Ruben].sql

```diff
@@ -0,0 +1,23 @@
+CREATE TABLE IF NOT EXISTS student(
+	id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    name VARCHAR(30) NOT NULL,
+    curp VARCHAR(18) NOT NULL UNIQUE,
+    count_number VARCHAR(10) NOT NULL UNIQUE
+);
+CREATE TABLE IF NOT EXISTS course(
+	id INT NOt NULL AUTO_INCREMENT PRIMARY KEY,
+    name VARCHAR(50) NOT NULL,
+    price DECIMAL(14,2)
+);
+CREATE TABLE IF NOT EXISTS teacher(
+	id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    name VARCHAR(30) NOT NULL,
+    topic VARCHAR(30),
+    courses_id INT NOt NULL,
+    FOREIGN KEY (courses_id) REFERENCES couse(id)
+);
+CREATE TABLE IF NOT EXISTS student_course(
+	students_id VARCHAR(30),
```

Comment:
Usualmente yo le pondría un constraint de NOT NULL a las columnas de las tablas intermedias, porque un registro sin estudiante o sin curso no tiene mucho sentido. También suelo ponerle el nombre de la tabla en singular para este caso.